base/applications/wordpad/wordpad.c

```diff
@@ -2294,6 +2294,7 @@ static LRESULT OnCommand( HWND hWnd, WPARAM wParam, LPARAM lParam)
         tr.chrg.cpMax = nLen;
         tr.lpstrText = data;
         SendMessageW(hwndEditor, EM_GETTEXTRANGE, 0, (LPARAM)&tr);
+        data[tr.chrg.cpMax - tr.chrg.cpMin] = L'\0';
         MessageBoxW(NULL, data, wszAppTitle, MB_OK);
         HeapFree( GetProcessHeap(), 0, data );
```

Comment:
While here, (document) why is it needed to allocate/get/display twice?

Reply:
Hm this means that when running this code, we get two successive messageboxes with stuff? weird.